apps/os/sdk/cli/index.ts

```diff
@@ -2,7 +2,12 @@ import { spawnSync } from "node:child_process";
 import { resolve } from "node:path";
 import { createCli } from "trpc-cli";
 import * as prompts from "@clack/prompts";
+import { proxify } from "trpc-cli/dist/proxify";
+import { createTRPCClient, httpLink } from "@trpc/client";
 import { testingRouter } from "../../backend/trpc/routers/testing.ts";
+import { appRouter } from "../../backend/trpc/root.ts";
```

Comment:
The appRouter contains more or less our entire backend. Do we really want to import that into our CLI? Now anyone writing any backend code needs to make sure it runs in our nodejs CLI or else `iterate trpc` breaks